README.md

```diff
@@ -125,7 +175,7 @@ urlretrieve(part_url, f'part-{part_id:06}.zip')
 shutil.unpack_archive(f'part-{part_id:06}.zip', f'part-{part_id:06}')
 ```
 
-### Method 3. Use `metadata.parquet` (Text Only)
+### Method 4. Use `metadata.parquet` (Text Only)
```

Comment:
Here should still be Method 3 :)

Reply:
Will change :D

Reply:
So that it still makes sense, I've removed the original Method 2 for manually downloading the data,